QtAwesome/iconic_font.py

```diff
@@ -85,42 +56,74 @@ def pixmap(self, size, mode, state):
         return pm
 
 
-class QtAwesome(QObject):
-    """The main class for managing icons"""
+_default_options = {
+    'color' : QColor(50, 50, 50),
+    'color-disabled' : QColor(70, 70, 70, 60),
+    'color-active' : QColor(10, 10, 10),
+    'color-selected' : QColor(10, 10, 10),
+    'scale-factor' : 0.9,
+}
+
+
+class IconicFont(QObject):
+    """The main class for managing iconic fonts"""
     
-    def __init__(self):
-        super(QtAwesome, self).__init__()
-        self.painter = QtAwesomeCharIconPainter()
+    def __init__(self, ttf_filename, codes):
+        """Takes a filename for the ttf font and a dictionary mapping icon
+        names to char numbers"""
+        super(IconicFont, self).__init__()
+        self.painter = CharIconPainter()
+        self.codes = codes
         self.painters = {}
-        _load()
+        self.ttf_name = ttf_filename
+        self._load_font()
+
+    def _load_font(self):
```

Comment:
In the latest version the `_load_font` is no longer "private" (`load_font`). I would leave it private as this is something that would not be directly exposed to the user... so better to have `_load_font`